src/Component/spec/Config/Builder/FieldSpec.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace spec\Sylius\Component\Grid\Config\Builder;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Grid\Config\Builder\Field;
+
+class FieldSpec extends ObjectBehavior
```

Comment:
```suggestion
final class FieldSpec extends ObjectBehavior
```
but that could be improved some day